File: app/src/main/java/cn/intret/app/picgo/model/RemoveFileMessage.java
package cn.intret.app.picgo.model;

import java.io.File;

/**
 * Remove file notification message
 */

public class RemoveFileMessage {
    private File mFile;

    public RemoveFileMessage setFile(File file) {
        mFile = file;
        return this;
    }

    public File getFile() {
        return mFile;
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/event/RemoveFileMessage.kt
package cn.intret.app.picgo.model.event

import java.io.File

/**
 * Remove file notification message
 */

class RemoveFileMessage {
    private lateinit var mFile: File

    fun setFile(file: File): RemoveFileMessage {
        mFile = file
        return this
    }

    fun getFile(): File {
        return mFile
    }
}
